systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -191,6 +197,141 @@ void testTieredStorageWithAivenPlugin() {
         MinioUtils.waitForNoDataInMinio(suiteStorage.getNamespaceName(), BUCKET_NAME);
     }
 
+    @ParallelTest
+    @TestDoc(
+        description = @Desc("This testcase is focused on testing of Tiered Storage integration implemented within Strimzi. The tests use the FileSystem plugin in Aiven Tiered Storage project (<a href=\"https://github.com/Aiven-Open/tiered-storage-for-apache-kafka/tree/main\">tiered-storage-for-apache-kafka</a>)."),
+        steps = {
+            @Step(value = "Deploys KafkaNodePool resource using combined KRaft node with PV of size 10Gi.", expected = "KafkaNodePool resource is deployed successfully with specified configuration."),
+            @Step(value = "Deploy Kafka CustomResource with Tiered Storage configuration pointing to another folder, using a built Kafka image. Reduce the `remote.log.manager.task.interval.ms` and `log.retention.check.interval.ms` to minimize delays during log uploads and deletions.", expected = "Kafka CustomResource is deployed successfully with optimized intervals to speed up log uploads and local log deletions."),
+            @Step(value = "Creates topic with enabled Tiered Storage sync with size of segments set to 10mb (this is needed to speed up the sync).", expected = "Topic is created successfully with Tiered Storage enabled and segment size of 10mb."),
+            @Step(value = "Starts continuous producer to send data to Kafka.", expected = "Continuous producer starts sending data to Kafka."),
+            @Step(value = "Wait until the remote folder size is greater than one log segment size (contains data from Kafka).", expected = "The remote folder contains at least one log segment from Kafka."),
+            @Step(value = "Wait until the earliest-local offset to be higher than 0.", expected = "The log segments uploaded to the remote folder are deleted locally."),
+            @Step(value = "Starts a consumer to consume all the produced messages, some of the messages should be located in remote folder.", expected = "Consumer can consume all the messages successfully."),
+            @Step(value = "Alter the topic config to retention.ms=10 sec to test the remote log deletion.", expected = "The topic config is altered successfully."),
+            @Step(value = "Wait until the remote folder is deleted.", expected = "The data in the remote folder are deleted.")
+
+        },
+        labels = {
+            @Label(value = TestDocsLabels.KAFKA)
+        }
+    )
+    void testTieredStorageWithAivenFileSystemPlugin() {
+        final TestStorage testStorage = new TestStorage(ResourceManager.getTestContext());
+
+        resourceManager.createResourceWithWait(
+            KafkaNodePoolTemplates.mixedPoolPersistentStorage(suiteStorage.getNamespaceName(), testStorage.getBrokerPoolName(), testStorage.getClusterName(), 1)
```

Comment:
By creating 1 combined node (controller + broker) only, we don't need to find the leader of the partition to verify the remote storage size in that node.

Reply:
I think it would be great to have this as a comment in the test